test/specs/dataUsageService.spec.js

```diff
@@ -20,14 +22,24 @@ describe('Data Usage Service', () => {
         dataUsageService = _dataUsageService_;
     }));
 
-
-
     it('should exist', function() {
         expect(dataUsageService).toBeDefined();
     });
 
     it('should be resolved', function() {
-        expect(dataUsageService.getDataUsage(clusterInterface)).not.toBe(undefined);
+        deferred.resolve([
+            {
+                name: "fakeWork"
+            },
+            {
+                name: "fakeHome"
+            }
+        ]);
+        $rootScope.$apply();
 
+        expect(dataUsageService.getDataUsage(clusterInterface)).not.toBe(undefined);
+        dataUsageService.getDataUsage(clusterInterface).then(function(data) {
```

Comment:
Make sure you do an expect on the spy to make sure that the clusterInterface function is called.